app/entities/users.py

```diff
@@ -14,73 +12,69 @@ def from_string(cls, string):
                 return User(int(string[3:-1]))
         return False
 
-    def __init__(self, id):
-        self.discord_id = id
-        
-        db_sess = db_session.create_session()
+    def __init__(self, uid):
+        self.discord_id = uid
+
+        db_sess = database.session()
         if not db_sess.query(SqlUser).filter(SqlUser.discord_id == self.discord_id
                                              ).first():
-            new = SqlUser(discord_id=id)
+            new = SqlUser(discord_id=uid)
             db_sess.add(new)
             db_sess.commit()
-        return None
 
     def __bool__(self):
         return True
 
-    async def DISCORD(self):
```

Comment:
wait, where did the `User.discord()` method go?

Reply:
usage of `ctx` in newly migrated modules allowed me to not use this function completely, and it's removal allowed me to prevent additional issues with recursive imports (this function required to import client directly, which isn't really possible, as `app` (where the client is located) imports the rest of the project)

Reply:
ok. still, this method seems useful, although I can't yet imagine how. Well, I'll figure that out on spot if I ever need this returned.